content/Solutions/Optimizations/Security.md

```diff
@@ -80,18 +82,12 @@ This improves connection stability between the share and the Apple system.
 When an administrators group is required, make sure the group members are correct.
 Administration group members have full permissions to modify or delete the share data.
 
-During [Share Creation]({{< relref "/CORE/CORETutorials/Sharing/SMB/SMBShare.md" >}}), a *Purpose* can be selected.
-This changes the share configuration with one click.
-For example, when selecting **Private SMB Datasets and Shares** from the list, TrueNAS automatically tunes some settings so the share is set up for private use.
-To fully customize the share settings, select **No presets** for the **Purpose**.
-Unless a specific purpose for the share is required, it is recommended to select **Default share parameters** as the **Purpose**.
-
 ### SSH
 
 Using Secure Shell (SSH) to connect to your TrueNAS is very helpful when issuing commands through the CLI.
-SSH settings are in **Services** after clicking the <span class="iconify" data-icon="mdi:pencil"></span> (pencil).
+SSH settings ([CORE SSH]({{< relref "ConfiguringSSH.md" >}}), [SCALE SSH]({{< relref "SSHServiceSCALE.md" >}})) are in **Services** after clicking the <span class="iconify" data-icon="mdi:pencil"></span> (pencil).
 
-For best security, disable the **Log in as Root with Password** and **Allow Password Authentication** SSH Service options.
+For best security, disable all login options for root or admin accounts in the SSH service options.
 Instead, create and exchange SSH keys between client systems and TrueNAS before attempting to connect with SSH.
 
 {{< hint type=warning >}}
```

Comment:
line 95 future tense, maybe make it this for active voice, present tense?
You cannot undo overwriting and SSH key pair.

Also, do you want to use tabs here or expands?